third_party/rust-on-exercism/health-statistics.rs

```diff
@@ -37,12 +37,9 @@ impl User {
             patient_name: &self.name,
             visit_count: self.visit_count as u32,
             height_change: measurements.height - self.height,
-            blood_pressure_change: match self.last_blood_pressure {
-                Some(lbp) => {
-                    Some((bp.0 as i32 - lbp.0 as i32, bp.1 as i32 - lbp.1 as i32))
-                }
-                None => None,
-            },
```

Comment:
We should probably exclude third_party/ somehow..

Reply:
Oh, good idea! Let me figure out how to do that before merging!

Reply:
If the settings are done in the workspace [Cargo.toml](https://github.com/google/comprehensive-rust/blob/ab78a9e6b86b17f370185596bb3cbc63a156b7dd/Cargo.toml) and every Cargo.toml has a `workspace = true` in the `[lints]` section as described in https://doc.rust-lang.org/cargo/reference/workspaces.html#the-lints-table we could tell cargo clippy what to do in the third_party Cargo.toml.

This would enable a central configuration for clippy and gives the option to override specific settings for individual components